pkg/universe.dagger.io/alpha/aws/cdk/python/command.cue

```diff
@@ -0,0 +1,143 @@
+package python
+
+import (
+	"dagger.io/dagger"
+	"dagger.io/dagger/core"
+	"universe.dagger.io/docker"
+)
+
+// Run a cdk command (`cdk <ARGS>')
+#Command: {
+	// Source code for infra
+	source: dagger.#FS
+
+	// Arguments to cdk
+	args: [...string]
+
+	// Project name, used for cache scoping
+	project: string | *"default"
+
+	_requirementsTxt: core.#Subdir & {
+		input: source
+		path:  "requirements.txt"
+	}
+
+	// requirements.txt file
+	requirementsTxt: core.#FS | *_requirementsTxt.output
+
+	// aws cdk cli package name
+	cdkCliName?: string
+
+	// aws cdk cli version
+	cdkCliVersion?: string
+
+	// Install
+	_install: #Install & {
+		requirementsTxt: requirementsTxt
+		"project":       project
+
+		if cdkCliName != _|_ {
+			"cdkCliName": cdkCliName
+		}
+
+		if cdkCliVersion != _|_ {
+			"cdkCliVersion": cdkCliVersion
+		}
+	}
+
+	// TODO: Use universe.dagger.op/aws.#Credentials once the package supports SSO and container credentials relative uri
+	aws: {
+		profile?:                         string
+		configFolder?:                    dagger.#FS
+		containerCredentialsRelativeUri?: string
+		accessKey?:                       string
+		secretKey?:                       dagger.#Secret
+	}
+
+	// Environment variables passed to the container where command is executed
+	env: [string]: string | dagger.#Secret
+
+	// Command logs
+	logs: container.export.files."/logs"
+
+	// Container where command is executed
+	container: docker.#Run & {
+		input:   _image.output
+		_image:  #Image
```

Comment:
Nit: `_image` should be above `input` since it's used in it because it's more readable.

As well, it should be possible to provide a custom definition, for instance if you want to use a different version than the default one in `#Image`?